src/components/sbb-radio-button/sbb-radio-button.scss

```diff
@@ -0,0 +1,87 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --sbb-radio-button-size: calc(var(--sbb-spacing-fixed-6x) - var(--sbb-border-width-1x) * 2);
+  --sbb-radio-button-label-color: var(--color-black-default);
+  --sbb-radio-button-checked-color: var(--sbb-color-red-default);
+  --sbb-radio-button-border-radius: var(--sbb-border-radius-8x);
+  --sbb-radio-button-transition: border var(--sbb-animation-duration-4x) ease;
+
+  // Align radio button to the first row of the label based on the line-height so that it's vertically
+  // aligned to the label and sticks to the top if the label breaks into multiple lines
+  --sbb-radio-button-icon-align: calc(
+    (
+        var(--sbb-font-size-text-m) * var(--sbb-typo-line-height-body-text) -
+          var(--sbb-radio-button-size)
+      ) / 2
+  );
+
+  display: inline-block;
+}
+
+:host(:focus-visible) {
+  --sbb-focus-outline-offset: var(--sbb-spacing-fixed-1x);
+  @include focusOutline;
+
+  border-radius: #{px-to-rem-build(0.25)};
+}
+
+:host([disabled]) {
+  --sbb-radio-button-label-color: var(--sbb-color-smoke-default);
+  --sbb-radio-button-checked-color: var(--sbb-color-graphite-default);
+}
+
+/* HTML5 Boilerplate accessible hidden styles */
+[type='radio'] {
+  @include screenReaderOnly;
+}
+
+/* One radio button per line */
+label {
+  @include text-m--regular;
+
+  cursor: pointer;
+  user-select: none;
```

Comment:
is user-select: none really needed?

Reply:
It essentially is like for the `sbb-toggle-check` where we decided to add the `user-select: none` given that you can click multiple time on the label to interact with the input and would be annoying to toggle the text selection on the label.